batavia/builtins/sum.js

```diff
@@ -1,7 +1,12 @@
+var operators = require('../operators')
 var exceptions = require('../core').exceptions
 var types = require('../types')
 var type_name = require('../core').type_name
 
+function add(left, right) {
+    return operators['__add__'].apply(left, right)
```

Comment:
Given this will be doing a dynamic attribute lookup on every `add`, would it be worth caching the attribute lookup as a module-level global? It wouldn't be the cleanest code, but it's one of those areas where a very small bit of dirt in the internals, could end up having a profound overall performance impact.

Reply:
Good call.